prow/cluster/sinker_rbac.yaml

```diff
@@ -17,6 +17,24 @@ rules:
     verbs:
       - delete
       - list
+      - watch
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    resourceNames:
+      - prow-sinker-leaderlock
```

Comment:
Why do we need this if we only have one replica of `sinker`?

Reply:
Upgrades

Reply:
If you think that warrants a separate discussion I will split it off this PR